source/device/cpu/cpu_module.c

```diff
@@ -295,7 +295,7 @@ int init_cpu_node_ops_registry(void)
 void release_cpu_node_ops_registry(void)
 {
     release_builtin_ops_registry();
-    release_custom_ops_registry();
+    // release_custom_ops_registry();
```

Comment:
why disable this? it seems no effect will be caused.

Reply:
because i didn't find the use of the init_custom_ops_registry function, commenting it out will avoid bug